### DISCUSSION TITLE
https://xaviw.github.io/XaviDocs/%E5%B7%A5%E5%85%B7%E7%B3%BB%E5%88%97/VitePress%E6%90%AD%E5%BB%BA/%E6%B7%BB%E5%8A%A0%E8%AF%84%E8%AE%BA%E5%8A%9F%E8%83%BD.html

### DISCUSSION THREAD

**Discussion by giscus** [Xaviw/XaviDocs#45] [Announcements]
# https://xaviw.github.io/XaviDocs/%E5%B7%A5%E5%85%B7%E7%B3%BB%E5%88%97/VitePress%E6%90%AD%E5%BB%BA/%E6%B7%BB%E5%8A%A0%E8%AF%84%E8%AE%BA%E5%8A%9F%E8%83%BD.html

个人前端技术文档

https://xaviw.github.io/XaviDocs/%E5%B7%A5%E5%85%B7%E7%B3%BB%E5%88%97/VitePress%E6%90%AD%E5%BB%BA/%E6%B7%BB%E5%8A%A0%E8%AF%84%E8%AE%BA%E5%8A%9F%E8%83%BD.html

**Comment by Xaviw** [comment]
欢迎评论🎇

**Comment by 7nami** [comment]
Hello

**Comment by zuoez02** [comment]
东西不错

**Comment by view21cy** [comment]
不知道有没有其他方式。QQ，微信登录？

**Reply by Xaviw** [reply]
giscus 只支持 GitHub 登录，评论数据也是存储在 Github 仓库中的。

QQ、微信等登录方式不借助服务器估计很难实现。

**Comment by RecklessZhang** [comment]
666

**Comment by qtanran** [comment]
我想把评论数据存到自己的数据库，不需要登录功能，请问有好的vitepress插件推荐吗

**Reply by Xaviw** [reply]
完整功能的插件确实没了解过，换个思路找个成熟的评论功能Vue组件，自己集成下应该会好找些

**Comment by NorthZeroD** [comment]
wow

**Comment by lhuans** [comment]
不错

**Comment by mtt-mtt** [comment]
456789哈哈哈可以

**Comment by xiaoshu312** [comment]
能不能在不同页面使用不同的语言？比如 (root) 使用 en，/zh/使用zh-CN这种的

**Reply by Xaviw** [reply]
4.0.0 版本已发布，完整支持 vitepress 的国际化、动态路由、路由重写功能，欢迎更新体验

**Comment by Xaviw** [comment]
插件 4.0.0 版本已发布，完整支持 vitepress 的国际化、动态路由、路由重写功能。欢迎更新体验，也欢迎提 issue、pr
